shopBuddy/shopBuddy/AddItemViewController.m

```diff
@@ -0,0 +1,62 @@
+//
+//  AddItemViewController.m
+//  shopBuddy
+//
+//  Created by Rachna Gupta on 7/12/22.
+//
+
+#import "AddItemViewController.h"
+#import "APIManager.h"
+#import "Item.h"
+#import "ItemDetailViewController.h"
+
+@interface AddItemViewController ()
+
+@end
+
+@implementation AddItemViewController
+
+- (void)viewDidLoad {
+    [super viewDidLoad];
+    [[APIManager shared] getItem:^(NSDictionary *itemDetails, NSError *error) {
+        NSLog(@"😎😎😎 Successfully loaded itemDetails");
+        if (itemDetails) {
+            NSLog(@"😎😎😎 Successfully loaded itemDetails");
+            self.item = [[Item alloc] initWithDictionary:itemDetails[@"products"][0]];
+            
+        }
+        else {
+            NSLog(@"😫😫😫 Error getting item details: %@", error.localizedDescription);
```

Comment:
You can add TODO comments that will help you remember things you need to revisit later.